recipes/libdis6/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.53.0"
+
+class libdis6Conan(ConanFile):
+    name = "libdis6"
+    homepage = "https://github.com/crhowell3/libdis6"
+    description = "Modern C++ implementation of IEEE 1278.1a-1998"
+    topics = ("library", "protocol", "dis")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "BSD-2-Clause"
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+
+    @property
+    def _min_cppstd(self):
+        return "14"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "15",
+            "msvc": "191",
+            "gcc": "7",
+            "clang": "7",
+            "apple-clang": "10",
+        }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.cache_variables["BUILD_EXAMPLES"] = False
+        tc.cache_variables["BUILD_TESTS"] = False
```

Comment:
```suggestion
        tc.cache_variables["BUILD_TESTS"] = False
        # CMake Error at cmake/libdis6-api.cmake:37 (set_property): could not find CACHE variable CMAKE_BUILD_TYPE
        tc.cache_variables["CMAKE_BUILD_TYPE"] = str(self.settings.build_type)
```

When building for Visual Studio, build type is not needed as we use `-C <config>` when building.